R/retrieve_credentials.R

```diff
@@ -0,0 +1,93 @@
+#' View available credentials keys in the system key-value store
+#'
+#' @export
+view_credential_keys <- function() {
+
+  get_keyring_creds_table() %>%
+    dplyr::select(-key_name) %>%
+    dplyr::rename(keyname = service)
```

Comment:
Add a column that contains the bare id (i.e. what you would pass to `creds_key`).

Reply:
Now changed to show `id`, `key_name`, and `username`.